### PR TITLE
Alternate results card style for tandem results

### PR DESCRIPTION
**Pull request by mjoyce91**
Can test by setting the defaultProp for `isTandem` or `isTandem2` to `true`.

### CODE REVIEW

File: src/sass/_results.scss [hunk 1]
```diff
@@ -55,6 +55,23 @@ $results-container-width: 100% - $filter-container-width;
     }
   }
 
+  &.results-card--tandem {
+    background-color: $primary-alt-lighter;
+
+    .compare-not-checked {
+      background-color: $color-white;
```
Comment (elizabeth-jimenez) [inline src/sass/_results.scss:62]: ⚪

File: src/sass/_results.scss [hunk 2]
```diff
@@ -55,6 +55,23 @@ $results-container-width: 100% - $filter-container-width;
...
+    }
+  }
+
+  .tandem-identifier {
+    font-size: 1.5rem;
+    font-weight: 500;
```
Comment (elizabeth-jimenez) [inline src/sass/_results.scss:68]: ⚖️

File: src/Components/ResultsCard/ResultsCard.jsx [hunk 3]
```diff
@@ -294,11 +325,15 @@ ResultsCard.propTypes = {
...
   favoritesPV: FAVORITE_POSITIONS_ARRAY,
+  isTandem1: PropTypes.bool,
+  isTandem2: PropTypes.bool,
 };
 
 ResultsCard.defaultProps = {
   favorites: [],
   favoritesPV: [],
+  isTandem1: false,
+  isTandem2: false,
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCard/ResultsCard.jsx:336]: 🎭

**Comment by mjoyce91** [comment]
>Was not adding any styling for results-card--tandem-two intentional?

@elizabeth-jimenez yes, one is just a placeholder in case we want to add styles in the future. However, I think I mixed them up - Tandem 2 is supposed to have the blue background

**Comment by elizabeth-jimenez** [comment]
> > Was not adding any styling for results-card--tandem-two intentional?
> 
> @elizabeth-jimenez yes, one is just a placeholder in case we want to add styles in the future. However, I think I mixed them up - Tandem 2 is supposed to have the blue background

Ah, cool 👍

**Pull request merged**